glfw/manifest

```diff
@@ -1,18 +1,18 @@
-: 1
+:1
 name: glfw
-version: 3.3.8+1
+version: 3.4.0
 summary: GLFW is an Open Source, multi-platform library for OpenGL, OpenGL ES and Vulkan application development. It provides a simple, platform-independent API for creating windows, contexts and surfaces, reading input, handling events, etc.
 license: Zlib
 description-file: README.md
 url: https://www.glfw.org
-package-url: https://github.com/Swat-SomeBug/glfw.git
-email: swat.somebug@gmail.com
-tests: glfw-tests == $
-examples: glfw-examples == $
-depends: * build2 >= 0.13.0
-depends: * bpkg >= 0.13.0
+package-url: https://github.com/monomere/glfw-build2.git
+email: monomere@duck.com
+# tests: glfw-tests == $
+# examples: glfw-examples == $
+depends: * build2 >= 0.16.0
```

Comment:
If updating here, we should update the other manifests as well; 

NOTE: stable is 0.17.0 now:

```suggestion
depends: * build2 >= 0.17.0
```